packages/expo-modules-core/ios/AppDelegates/ExpoAppDelegate.swift

```diff
@@ -21,6 +21,17 @@ open class ExpoAppDelegate: UIResponder, UIApplicationDelegate {
   // MARK: - Initializing the App
 
   open func application(_ application: UIApplication, willFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey : Any]? = nil) -> Bool {
+    let isWillFinishLaunchingWithOptionsImplmented = subscribers.contains {
```

Comment:
```suggestion
    let isWillFinishLaunchingWithOptionsImplemented = subscribers.contains {
```

Also, wouldn't it be more clear if we filter the subscribers here and then check if it's empty or reduce otherwise? 🤔